doc/remediation-templating.md

```diff
@@ -93,305 +40,204 @@ spec:
 
-`compliance.openshift.io/unset-value`: We used this annotation to mark all the XCCDF variables that are successfully parsed, but are not found in `ResultConfigMap` or there the value is in `compliance.openshift.io/value-required` annotation, but the tailored value for that value was not set by the administrator.
+## Additional features: Value-Required annotation.
 
-`compliance.openshift.io/xccdf-value-used`: This one included all the values that were initially parsed as well as value set from `value-required`
+We have `kubernetes_machine_config_file_with_required_value(path='', file_permissions_mode='', source='', vals=[])`
+A content writer can put variable name in `val` to force an administrator to set a value for that variable before they can 
+successfully apply the remediation.
 
-The remediation status will become `NeedsReview` where if there is `unset-value`
-If the the Remediation Status become `Needs-Review`, the administrator can check for the annotation `compliance.openshift.io/has-unset-xccdf-values` for that `ComplianceRemediation`,
-because some rules require to have a custom value to be set, or it didn't come with a default value. The Following is a example of missing value ComplianceRemediation object look like.
+It will set an annotation for machine config remediation: `complianceascode.io/value-input-required:` 
 
-```yaml
-apiVersion: compliance.openshift.io/v1alpha1
-kind: ComplianceRemediation
-metadata:
-  creationTimestamp: '2021-07-01T19:06:43Z'
-  annotations: 
-    compliance.openshift.io/has-unset-xccdf-values: "sshd_idle_timeout_value"
-  labels:
-    compliance.openshift.io/has-unset-variable: "sshd_idle_timeout_value"
-```
 
-#### Content writer user-flow
+## Administrator user-flow
 
-As a content writer:
-It is now possible to write remediation content for Kubernetes and use the XCCDF values from
-the rules file.
+Here are some annotations and labels we used for the compliance remediation:
+ 
+`compliance.openshift.io/value-required`: we used this one for the content creator to mark XCCDF variables to be required, and an administrator who runs the scan needs to set a value for that XCCDF variables, or else it will show as `NeedsReview` for that remediation.
+ 
+`compliance.openshift.io/unset-value`: We used this annotation to mark all the XCCDF variables that are not found in `ResultConfigMap` or when there are values in `compliance.openshift.io/value-required` annotation, but those values are not found in the tailored profile.
+ 
+`compliance.openshift.io/xccdf-value-used`: This one included all the values that were initially parsed as well as the value set from `compliance.openshift.io/value-required`.
+ 
+RemediationUnsetValueLabel `compliance.openshift.io/has-unset-variable`: we use this one to label remediation that has unset variables
```

Comment:
Looks like the `RemediationUnsetValueLabel` name is a typo brought over from CO sources?

Reply:
sorry, I forgot to delete `RemediationUnsetValueLabel `, is there a typo here?